bedrock/firefox/urls.py

```diff
@@ -62,6 +62,8 @@
          ftl_files=['firefox/features/shared', 'firefox/features/private-browsing']),
     page('firefox/features/safebrowser', 'firefox/features/safebrowser.html'),
     url(r'^firefox/features/translate/$', views.firefox_features_translate, name='firefox.features.translate'),
+    page('firefox/features/picture-in-picture', 'firefox/features/picture-in-picture.html',
```

Comment:
suggestion/nit: Should we make a redirect for `firefox/features/pip`?

Reply:
Added!